pkgs/top-level/impure.nix

```
@@ -37,7 +37,11 @@ in
   # collections of packages.  These collection of packages are part of the
   # fix-point made by Nixpkgs.
   overlays ? let
-      isDir = path: builtins.pathExists (path + "/.");
+      isDir = path: let
+        parent = dirOf path;
+        base = baseNameOf path;
+        type = (builtins.readDir parent).${base} or null;
+      in (/. + path) == /. || type == "directory";
```

```suggestion
      isDir = path: builtins.pathExists (path + "/");
```

We can just remove the `.` for now. Terrible, but it should work